nix/pkgs/neard/stable.nix

```
@@ -4,7 +4,7 @@ let
   generic = pkgs.callPackage ./generic.nix { };
 in
 generic {
-  version = "1.29.0";
-  sha256 = "sha256-TOZ6j4CaiOXmNn8kgVGP27SjvLDlGvabAA+PAEyFXIk=";
-  cargoSha256 = "sha256-LFYWkQY7UcFg0aImfS3cWGKviRdG+gP9Vv2QUZgxtsg=";
+  version = "1.30.0-rc.4";
+  sha256 = "sha256-0J6dJf/OJGL8avhIKIs1HlfsCi46B+Wv810qMQQtp3w=";
+  cargoSha256 = "sha256-X4hU9rqHQ9gcT2rQaKhY9o3flVZDO2w7gM8s6GQBHec=";
```

I think this was miss communication. We should stay on stable here and only use `neardUnstable` when testing
our patch:
 
```suggestion
  version = "1.29.0";
  sha256 = "sha256-TOZ6j4CaiOXmNn8kgVGP27SjvLDlGvabAA+PAEyFXIk=";
  cargoSha256 = "sha256-LFYWkQY7UcFg0aImfS3cWGKviRdG+gP9Vv2QUZgxtsg=";
```

---

Got it.